src/transformers/models/owlvit/feature_extraction_owlvit.py

```diff
@@ -109,18 +108,19 @@ def post_process(self, outputs, target_sizes):
             `List[Dict]`: A list of dictionaries, each dictionary containing the scores, labels and boxes for an image
             in the batch as predicted by the model.
         """
-        out_logits, out_bbox = outputs.logits, outputs.pred_boxes
+        logits, boxes = outputs.logits, outputs.pred_boxes
```

Comment:
This seems quite similar to DETR's one. Can we leverage `Copied from` statements here?

Reply:
I actually noticed there were a few bugs in the post_process code. I fixed them but we can't use the copied from statements anymore

Reply:
Feel free to correct them for DETR as well :)

Reply:
I meant it was copied from DETR but didn't work for OWL-ViT because it is not a multi-class classification problem, each text query input is evaluated in a binary manner.

DETR's `post_process()` method works

Reply:
Ok